app/assets/src/components/views/discovery/mapping/base_map.scss

```diff
@@ -10,8 +10,12 @@
     position: absolute;
     bottom: 50px;
     right: 15px;
+    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.1);
```

Comment:
If we can't reuse a box-shadow already defined in `_elements` (always double check with Jenn to push for consistency), move this definition there and use a variable here.

Reply:
changed 👍